glslc/test/working_directory.py

```diff
@@ -207,11 +157,63 @@ class TestWorkDirCompileFileAbsolutePath(expect.ValidObjectFile):
     glslc_args = ['-c', '-working-directory=subdir', shader]
 
 
-@inside_glslc_testsuite('WorkDir')
-class TestWorkDirLinkFileAbsolutePath(expect.ValidNamedObjectFile):
-    """Tests -working-directory=<dir> when linking input file with absolute
-    path."""
+# The -working-directory flag should not affect the placement of the link file.
+# The following tests ensure that.
 
-    shader = FileShader('void main() {}', '.vert')
-    glslc_args = ['-working-directory=subdir', shader]
+class WorkDirDoesntAffectLinkedFile(expect.ValidNamedObjectFile):
+    """A base class for tests asserting that -workind-directory has no impact
+    on the location of the output link file.
+    """
+    environment = Directory('.', [
+        Directory('subdir', [
+            File('shader.vert', 'void main(){}'),
+            # This bin/ is in the wrong location, so glslc should ignore it.
```

Comment:
I'd suggest to reword this sentence to something like "This bin/ is not where linked files are generated...". "In the wrong location" sounds strange for me. We are freely to create a bin/ directory anywhere.